lib/pivotal-tracker/account_membership.rb.orig

```diff
@@ -0,0 +1,31 @@
+class PivotalTracker::AccountMembership
+  attr_accessor :kind, :id, :person, :account_id, 
+                :created_at, :updated_at, :owner, :admin,
+                :project_creator, :timekeeper, :time_enterer
+
+  def initialize(membership_attributes)
+    membership_attributes.each do |attribute, value|
+      self.send(:"#{attribute}=", value)
+    end
+  end
+
+  def self.all(account_id)
+    response = PivotalTracker.get("/accounts/#{account_id}/memberships")
+    if response.code == 200
+      response.parsed_response.map do |membership|
+        PivotalTracker::AccountMembership.new(membership)
+      end
+    else
+      puts response.inspect
+    end
+  end
+
+  def self.create(membership_attributes, owner_id)
+    response = PivotalTracker.post("/accounts/#{owner_id}/memberships", query: membership_attributes)
+<<<<<<< HEAD
```

Comment:
@wlffann merge conflicts! I see this has a `.orig` extension. Maybe just delete the files with that extension?